dev/NavigationView/NavigationView.h

```diff
@@ -34,6 +34,10 @@ enum class NavigationRecommendedTransitionDirection
     Default // Currently it's mapping to EntranceNavigationTransitionInfo and is subject to change.
 };
 
+static constexpr auto c_topNavRepeater = L"TopNavMenuItemsHost"sv;
```

Comment:
>c_topNavRepeater [](start = 22, length = 16)

i think most of these are s_ variables

Reply:
I moved and integrated them in the other similar variables defined in the cpp file

---
In reply to: [353458662](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r353458662) [](ancestors = 353458662)